policies/omc-bylaws.md

```diff
@@ -0,0 +1,341 @@
+# OpenSSL Bylaws
```

Comment:
Even thought this isn't a material change, I suspect an OMC bylaw vote will be required.

Reply:
So on the next OMC meeting I guess?

Reply:
Suggest adding the trivial edits text from the policies:

> # Minor Edits
> 
> Minor bylaw edits that do not change the meaning of the edited bylaws do not require this voting process. Typical examples of such edits are spelling, grammar, and formatting fixes.
> 
> These edits are done via pull requests that are approved by two OMC members where neither of them is the author of the submission. The pull request should be labelled with the minor edit label.
> 
> Approved submissions shall only be applied after a 24-hour delay from the approval.

Reply:
Added...  I assume it was the right move to make it into this PR, right?

Reply:
Yep.